client/scss/layouts/_home.scss

```diff
@@ -1,31 +1,6 @@
-.homepage header {
-  @apply w-text-primary;
-
-  display: flex;
-  flex-flow: row nowrap;
-  gap: 1em;
-
-  .col1 {
-    width: 50px;
-    margin-inline-end: 1em;
-    padding: 0;
-
-    // make way for the nav-menu button on mobile
-    @include media-breakpoint-down(xs) {
-      position: relative;
-      inset-inline-start: $mobile-nav-indent;
-    }
-  }
-
-  .avatar {
-    margin-top: 0.5em;
-    flex-shrink: 0;
-  }
-
-  .user-name {
-    font-size: 1.3em;
-    font-weight: 600;
-  }
+.homepage .user-name {
+  font-size: 1.3em;
+  font-weight: 600;
```

Comment:
it would be nice to not need this styling either - if using the subtitle variant we may be fine OR we may need some of these styles still.